ansible/playbooks/dev/app_python/main.yml

```diff
@@ -0,0 +1,14 @@
+- hosts: vms
+  pre_tasks:
+    - name: Update apt cache
+      apt:
+        update_cache: yes
+      become: true
+
+  roles:
+    - name: Deploy the docker container
+      role: deploy_docker
+      vars:
+        image_name: ntdesmond/iu-devops-python
+        internal_port: 8000
+        external_port: 80
```

Comment:
I don't see tags here 🤔

Reply:
> I don't see tags here 🤔

Why would I need them, though?

Reply:
I mean, they exist in roles, but I don't use them

Reply:
To call only wipe-0 tasks for example

Reply:
> To call only wipe-0 tasks for example (& 'cause it's required in task LOL)

Well if one needs that they can call them. I've added them in the role itself, but in the playbook I do not need a separate wipe without further installation.